python/src/aiconfig/editor/client/src/components/AIConfigEditor.tsx

```diff
@@ -931,134 +935,143 @@ export default function EditorContainer({
   const runningPromptId: string | undefined = aiconfigState._ui.runningPromptId;
 
   return (
-    <AIConfigContext.Provider value={contextValue}>
-      <Notifications />
-      {serverStatus !== "OK" && (
-        <>
-          {/* // Simple placeholder block div to make sure the banner does not overlap page contents until scrolling past its height */}
-          <div style={{ height: "100px" }} />
-          <Alert
-            color="red"
-            title="Server Connection Error"
-            w="100%"
-            style={{ position: "fixed", top: 0, zIndex: 999 }}
-          >
-            <Text>
-              There is a problem with the editor server connection. Please copy
-              important changes somewhere safe and then try reloading the page
-              or restarting the editor.
-            </Text>
-            <Flex align="center">
-              <CopyButton
-                value={JSON.stringify(
-                  clientConfigToAIConfig(aiconfigState),
-                  null,
-                  2
+    <AIConfigEditorThemeProvider mode={mode}>
+      <AIConfigContext.Provider value={contextValue}>
+        <Notifications />
+        <div className="editorBackground">
```

Comment:
is this className=editorBackground automatically available when `AIConfigEditorThemeProvider` is used?

Reply:
Yes, as long as the themes define it. The mantine provider will put all the theme-defined classes into global css